Frontend/src/Components/Faults/FaultsView.js

```diff
@@ -381,55 +367,55 @@ export default function Faults(props) {
           <Image
             fit={fitType}
             boxSize={imageHeight}
-            src={BatteryFailsafeImage}
+            src={Images.BatteryFailsafe}
           />
         </Tooltip>
       ) : (
         <Box h={imageHeight} />
       )}
       {_checkBooleanData("bps_fault") ? (
         <Tooltip label={"BPS fault"}>
-          <Image fit={fitType} boxSize={imageHeight} src={BPSFaultImage} />
+          <Image fit={fitType} boxSize={imageHeight} src={Images.BPSFault} />
         </Tooltip>
       ) : (
         <Box h={imageHeight} />
       )}
       {props.data?.soc[0] < CONSTANTS.soc.MIN + 5 ? (
-        <Tooltip label={"Battery charge is low (<5%)"}>
-          <Image fit={fitType} boxSize={imageHeight} src={LowBatteryImage} />
+        <Tooltip label={"Battery charge is low (<" + (CONSTANTS.soc.MIN + 5) + "%)"}>
```

Comment:
It was meant as a warning to indicate that you're getting close to your nominal minimum value for SoC (like a low fuel light on your car), but I guess it would make more sense to just adjust the nominal min in the format to this "warning value"